# Parameters

run_derivative_test: false

# Possible gaits: 
# 0 (C0 - overlap-walk)
# 1 (C1 - running trot)
# 2 (C2 - running pace) 
# 3 (C3 - bounding) 
# 4 (C4 - gallop) 
# 5 (C5 - static-walk) 
# 6 (C6 - overlap-walk)
# 7 (lateral troting)
# 8 (driving)
# 9 (BlockCross - gallop) => works well for 3 polynomials in the swing phase
# 10 (BlockBound - bounding) 
# 11 (GapCross - gallop) => works well for 3 polynomials in the swing phase
# 12 (GapBound - bounding)  
# 13 (BlockRight)

constrain_final_z_base: false
n_polys_in_swing_phase: 3 #2
min_distance_above_terrain: 0.0 #0.08

Flat:
  use_non_holonomic_constraint: false
  limit_base_angles: false
  max_wheels_acc_z: 50.0

Gap:
  use_non_holonomic_constraint: false
  limit_base_angles: true # generates smoother motions, not mandatory
  max_wheels_acc_z: 40.0
# obs: for the gallop gait, reducing max z-acc for the base improves the motion a little (optional)

Block:
  use_non_holonomic_constraint: false
  limit_base_angles: false
  max_wheels_acc_z: 40.0

BlockRight:
  use_non_holonomic_constraint: false
  limit_base_angles: false
  max_wheels_acc_z: 40.0

Steps:
  use_non_holonomic_constraint: false
  limit_base_angles: false
  max_wheels_acc_z: 40.0






